code/datums/components/butchering.dm

```diff
@@ -138,7 +141,14 @@
 /datum/component/butchering/recycler
 
 
-/datum/component/butchering/recycler/Initialize(_speed, _effectiveness, _bonus_modifier, _butcher_sound, disabled, _can_be_blunt)
+/datum/component/butchering/recycler/Initialize(
+	speed,
+	effectiveness,
+	bonus_modifier,
+	butcher_sound,
+	disabled,
+	can_be_blunt
```

Comment:
trailing comma please, and make sure the args are the same as parent

Reply:
done